packages/tc-schema-validator/validation/json-schema/type.js

```diff
@@ -200,6 +200,7 @@ const fromOrTo = {
 	properties: {
 		type: { type: 'string', enum: types },
 		hasMany: { type: 'boolean' },
+		otherNodeName: {type: 'string'},
```

Comment:
Should we make this more strict?

Reply:
yep good point. It'll need to use the same regex pattern as propert names do

Reply:
Added `pattern: PROPERTY_NAME` validation.